api.py

```diff
@@ -15,28 +15,27 @@ def index():
 def getalltask():
     task_json = get_alltask()
     if task_json:
-        task_return = jsonify(task_json)
+        result = jsonify(task_json)
     else:
-        task_return = jsonify({"message": "タスクデータはありません"})
-    return task_return
+        result = "タスクデータはありません"
+    return result
 
 
-@app.route('/task/<n>', methods=['GET'])
-def gettask(n):
-    task_json = get_task(n)
+@app.route('/task/<id>', methods=['GET'])
+def gettask(id):
+    task_json = get_task(id)
     if task_json:
-        task_return = jsonify(task_json)
+        result = jsonify(task_json)
     else:
-        task_return = jsonify({"message": "該当idのデータはありません"})
-    return task_return
+        result = "該当idのデータはありません"
+    return result
 
 
 @app.route('/task', methods=['POST'])
 def posttask():
     post_json = request.get_data()
-    post_task(post_json)
-    complete = "COMPLETE"
-    return complete
+    result = post_task(post_json)
```

Comment:
resultが中間変数になってしまっているので

```suggestion
    return post_task(post_json)
```
　でいいかな